### PR TITLE
Update the document to show how to uninstall thin-edge.io

### PR DESCRIPTION
**Pull request by PradeepKiruvale**
## Proposed changes

This PR captures the documentation updates to describe how to use the `uninstall-thin-edge.io` script.

## Types of changes

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Improvement (general improvements like code refactoring that doesn't explicitly fix a bug or add any new functionality)
- [x] Documentation Update (if none of the other choices apply)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Paste Link to the issue
<br/>

## Checklist

- [x] I have read the [CONTRIBUTING](https://github.com/thin-edge/thin-edge.io/blob/main/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://github.com/thin-edge/thin-edge.io/blob/main/CONTRIBUTOR-LICENSE-AGREEMENT.md) (in all commits with git commit -s)
- [ ] I ran `cargo fmt` as mentioned in [CODING_GUIDELINES](https://github.com/thin-edge/thin-edge.io/blob/main/CODING_GUIDELINES.md)
- [ ] I used `cargo clippy` as mentioned in [CODING_GUIDELINES](https://github.com/thin-edge/thin-edge.io/blob/main/CODING_GUIDELINES.md)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: docs/src/howto-guides/002_installation.md [hunk 1]
```diff
@@ -131,6 +131,43 @@ Eg:
 dpkg -i tedge_mapper_0.5.0_armhf.deb
 ```
 
+## Uninstall `thin-edge.io`
+The `thin-edge.io` can be uninstalled using a script, the script can be downloaded
+from below mentioned location. 
+
+```shell
+https://raw.githubusercontent.com/thin-edge/thin-edge.io/main/uninstall-thin-edge_io.sh
```
Comment (albinsuresh) [inline docs/src/howto-guides/002_installation.md:139]: Shouldn't this be a `curl` or `wget` command?

File: docs/src/howto-guides/002_installation.md [hunk 2]
```diff
@@ -131,6 +131,43 @@ Eg:
...
+USAGE:
+    delete-thin-edge_io [COMMAND]
+    
+COMMANDS:
+    remove     Uninstall thin-edge.io with keeping configuration files
+    purge      Uninstall thin-edge.io and also remove configuration files
+```
+
+> Note: The uninstall script removes/purges the core thin-edge.io packages like `tedge, tedge_mapper, and tedge_agent`.
+> It also removes/purges the thin-edge.io plugins like `tedge_apt_plugin, c8y_log_plugin, and c8y_configuration_plugin`.
```
Comment (albinsuresh) [inline docs/src/howto-guides/002_installation.md:153]:
```suggestion
> Note: The uninstall script removes/purges the core thin-edge.io packages like `tedge, tedge_mapper, and tedge_agent` as well as thin-edge.io plugins like `tedge_apt_plugin, c8y_log_plugin, c8y_configuration_plugin` etc.
```

File: docs/src/howto-guides/002_installation.md [hunk 3]
```diff
@@ -131,6 +131,43 @@ Eg:
 dpkg -i tedge_mapper_0.5.0_armhf.deb
 ```
 
+## Uninstall `thin-edge.io`
+The `thin-edge.io` can be uninstalled using a script, the script can be downloaded
```
Comment (albinsuresh) [inline docs/src/howto-guides/002_installation.md:135]:
```suggestion
The `thin-edge.io` can be uninstalled using a script, that can be downloaded
```

**Review by albinsuresh — APPROVED** [review]

File: docs/src/howto-guides/002_installation.md (hunk 1 above)
Comment (PradeepKiruvale) [reply docs/src/howto-guides/002_installation.md:139]: Yup, fixed it.

File: docs/src/howto-guides/002_installation.md (hunk 2 above)
Comment (PradeepKiruvale) [reply docs/src/howto-guides/002_installation.md:153]: fixed

File: docs/src/howto-guides/002_installation.md (hunk 3 above)
Comment (PradeepKiruvale) [reply docs/src/howto-guides/002_installation.md:135]: fixed

**Pull request merged**